lib/Config.php

```diff
@@ -342,6 +343,8 @@ public function getTurnServers(bool $withEvent = true): array {
 		if ($withEvent) {
 			$event = new GetTurnServersEvent($servers);
 			$this->dispatcher->dispatchTyped($event);
+			$event = new BeforeTurnServersGetEvent($event->getServers());
+			$this->dispatcher->dispatchTyped($event);
```

Comment:
Are those not the same thing?

Reply:
`GetTurnServersEvent` is not following the suggested naming schema from https://docs.nextcloud.com/server/latest/developer_manual/basics/events.html#naming-scheme
So replaced it with a "valid" one :)

Reply:
Would it work to have one extend the other so we only dispatch once or does listener not work that way?

Reply:
the endpoint is only called once per visitor and I will drop the event once the internal listeners have been migrated, so I don't think it's worth any shenanigans